internals/webpack/webpack.base.babel.js

```diff
@@ -88,4 +88,9 @@ module.exports = (options) => ({
   target: 'web', // Make web variables accessible to webpack, e.g. window
   stats: false, // Don't show stats in the console
   progress: true,
+
+  // Polling for vagrant based development
```

Comment:
since polling is not necessary for i18n and is CPU-intensive, maybe this should be moved into separate pr... i don't use vagrant/docker, but i understand this is an important feature for those environments, so maybe this could be another PR but moving the poll value into config but leaving default as false? @mxstbr thoughts?